python/apps/taiga/src/taiga/users/services/__init__.py

```diff
@@ -225,6 +228,33 @@ async def update_user(user: User, full_name: str, lang: str) -> User:
     return updated_user
 
 
+#####################################################################
+# delete user
+#####################################################################
+
+
+# TODO
+
+
+#####################################################################
+# delete info user
+#####################################################################
+
+
+async def get_user_delete_info(user: User) -> UserDeleteInfoSerializer:
+    ws_list = await _list_workspaces_delete_info(user=user)
+    pj_list = await _list_projects_delete_info(user=user, ws_list=ws_list)
+
+    ws_list_serialized = [
+        serializers_services.serialize_workspace_with_projects_nested(
+            workspace=workspace, projects=await workspaces_repositories.list_workspace_projects(workspace=workspace)
+        )
+        for workspace in ws_list
+    ]
+
+    return UserDeleteInfoSerializer(workspaces=ws_list_serialized, projects=pj_list)
```

Comment:
I think this should go in a service serailizer, like in stries reorder endpoint (for example)

```
def serialize_reorder_story(
    status: WorkflowStatus, stories: list[int], reorder: dict[str, Any] | None = None
) -> ReorderStoriesSerializer:

    return ReorderStoriesSerializer(status=status, stories=stories, reorder=reorder)
```

Reply:
We only create a service serializer if we have a manytomany or is not automatic, this is the convention.
In this case we don't need it.